main/src/components/userLogin.js

```diff
@@ -24,22 +25,13 @@ class UserLogin extends React.Component {
   };
 
   async handleSubmits(e) {
-    console.log("Form submitted with the following data");
-    console.log(this.state.pwd);
     e.preventDefault();
-
-    // const res = await axios.get("/login", {
-    //   params: {
-    //     user_email: `${this.state.email}`,
-    //     user_password: `${this.state.pwd}`,
-    //   },
-    // });
-
     const res = await axios.post("/login", {
       user_email: `${this.state.email}`,
       user_password: `${this.state.pwd}`,
     });
-    console.log(res);
+    this.state.loggedIn = true;
+    localStorage.setItem('user', JSON.stringify(res.data))
```

Comment:
Will need to double check what is exactly is being stored in localStorage and how exactly we can get the user info from it. Other than that the code looks good.